src/camera.cpp

```diff
@@ -471,14 +474,64 @@ void Camera::update(LocalPlayer* player, f32 frametime, f32 busytime, f32 tool_r
 	if (m_arm_inertia)
 		addArmInertia(player->getYaw());
 
+	// New: via SplineSequences
+	std::map<std::string, SplineSequence<v3f> > positionsplines;
```

Comment:
i don't think order is important, unordered_map ?

Reply:
True; will change.